src/lib.rs

```diff
@@ -249,8 +245,10 @@ pub trait Watcher: Clone {
     /// that was set on the watchable before it was dropped.
     fn get(&mut self) -> Self::Value;
 
+    fn to_latest(&mut self) -> bool;
```

Comment:
what exactly does this do?

Reply:
I've renamed `to_latest` to `update`, given that's the terminology we're using everywhere else (`updated`, `poll_updated`, etc.).
There's a bit of a chance of confusing `.update()` and `.updated()` as a user, but I still think that's the best term to use.